### ISSUE TITLE
CR8 Back end Dest Tag

### ISSUE THREAD

**Issue by Mattch33** [asgiom/cr8-brm-mobile-issues#63]
We need the ability to override the greyed out boxes in the back end so that we can attach data. 

For example:

Users acc has not been activated correctly with casino and the Destination tag has not been picked up. We can reach out to them as support and they can tell us the tag so we can manually insert.

**Comment by danielwwf** [comment]
Just to confirm: Admins should be able to overwrite every setting in both backends no matter what. There will be only few who can do so anyways.

**Comment by t3nshi** [comment]
This was already implemented... does it comply to the requirement?

**Comment by Mattch33** [comment]
No

**Comment by danielwwf** [comment]
It doesnt allow manual changes

![image](https://user-images.githubusercontent.com/30388756/93606914-f062be80-f9c8-11ea-9af9-bf019583bdb9.png)

**Comment by t3nshi** [comment]
Let's make editable the fields that make sense to edit... but not everything... for example here we could edit the name of the operator and the destinationtag... even the operator accountID (if it was manual maybe was set wrong) (taking into account that if put wrong the deposit wont work), but the useraccountid for the operator should remain as it is not changing for a user... the registration date also shouldn't be editable... do you agree?

**Comment by danielwwf** [comment]
Yes, the superuser needs permission to edit everything that's dynamic, like the desttag but not stuff that is set by other settings on the backend, like the operatorID

**Comment by t3nshi** [comment]
great. We will try to push this today.
